src/translator/sentence_ranges.cpp

```diff
@@ -99,5 +99,42 @@ string_view AnnotatedText::asStringView(const ByteRange &byteRange) const {
   return string_view(data, size);
 }
 
+string_view AnnotatedText::pre(size_t sentenceIdx) const {
+  // Find start of filler-text before, there's a corner case when there's no
+  // sentence before.
+  const char *start{nullptr};
```

Comment:
the C++ style is usually = nullptr;

Reply:
Why are we even initializing if both branches of if initialize it?

Reply:
Removed brackets initialization altogether.